vertx-mongo-client/src/main/java/io/vertx/ext/mongo/MongoClient.java

```diff
@@ -262,10 +263,8 @@ MongoClient bulkWriteWithOptions(String collection, List<BulkOperation> operatio
    *
    * @param collection  the collection
    * @param query  query used to match documents
-   * @param resultHandler  will be provided with each found document
    */
-  @Fluent
-  MongoClient findBatch(String collection, JsonObject query, Handler<AsyncResult<JsonObject>> resultHandler);
+  ReadStream<JsonObject> findBatch(String collection, JsonObject query);
```

Comment:
This is a breaking change, so we need to mention it in the breaking change list from 3.5.0 to 3.5.1.

Reply:
👍